src/main/java/com/cottonfactory/products/controllers/AccessoriesController.java

```diff
@@ -0,0 +1,25 @@
+
+package com.cottonfactory.products.controllers;
+
+import com.cottonfactory.products.entities.Accessories;
+import com.cottonfactory.products.services.AccessoriesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class AccessoriesController {
+
+    @Autowired
+    AccessoriesService accessoriesService;
```

Comment:
is it possible to use constructor injection instead of field injection? and the service is better as private and/or final